ext_mongodb.php

```diff
@@ -620,6 +626,94 @@ class BulkWriteException extends WriteException {}
 /* {{{ BSON and Serialization Classes */
 namespace MongoDB\BSON;
 
+/* {{{ Interfaces */
+interface BinaryInterface
+{
+	public function serialize() : string;
+	public function unserialize(mixed $data) : void;
```

Comment:
No, the interface should extend Serializable, which should make your second point go away too?